cymetric/root_metrics.py

```diff
@@ -34,6 +39,7 @@ def __call__(self, conds=None, *args, **kwargs):
 
 
 def root_metric(obj=None, name=None, schema=None, *args, **kwargs):
+    """Checks for existence of root metric."""
```

Comment:
Same decorator idea as with `metric()`. Please see "Effective Computation in Physics" for a description of decorators if you need one :).